Source/WebCore/css/counterStyles.css

```
@@ -263,7 +265,8 @@
 
 @counter-style square {
     system: cyclic;
-    symbols: \25AA;
+    /*  CSS specification calls for U+25FE BLACK MEDIUM SMALL SQUARE; we have been using U+25A0 BLACK SQUARE instead for a long time. */
```

```suggestion
    /* CSS specification calls for U+25FE BLACK MEDIUM SMALL SQUARE; we have been using U+25A0 BLACK SQUARE instead for a long time. */
```
nit: small extra space